src/seedu/addressbook/data/person/Address.java

```diff
@@ -26,10 +30,16 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
         this.value = address;
+        String[] addressSections = address.split(", ");
```

Comment:
as discussed in the tutorial, no need to save the input address in `this.value` as it may cause inconsistency due to duplication.